<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOTriggerWidget</class>
 <widget class="QWidget" name="IOTriggerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>801</width>
     <height>661</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>IO触发配置</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>581</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>IO1触发测试</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>触发类型:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="IO1TriggerTypeCbB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>51</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="IO1TestButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>42</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>测试</string>
      </property>
     </widget>
     <widget class="QLabel" name="IO1TestResultLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>没有触发信号</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>581</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>IO2触发测试</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>触发类型:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="IO2TriggerTypeCbB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>51</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="IO2TestButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>42</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>测试</string>
      </property>
     </widget>
     <widget class="QLabel" name="IO2TestResultLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>没有触发信号</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>IO1触发设置</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>54</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>延时:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>滤波:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="IO1DelayMsEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="IO1SmoothingEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>79</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>36</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>82</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>400</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>IO2触发设置</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>54</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>延时:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>滤波:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="IO2DelayMsEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="IO2SmoothingEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>79</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>36</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>82</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>触发设定</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>任务列表</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IO触发端口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>触发方式</string>
     </property>
    </widget>
    <widget class="QListWidget" name="TaskTriggerSetListWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>51</y>
       <width>781</width>
       <height>570</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
